engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -193,31 +162,26 @@ public boolean pregenerateChunks() {
     }
 
     @Override
-    public void update(float delta) {
+    public void update(float deltaInMs) {
 
         PerformanceMonitor.startActivity("Update Tick");
```

Comment:
I have been puzzled for some time about these PerformanceMonitor statements. This one in particular is a bit strange as I doubt the updateTick() method is of any relevance from a Performance perspective, as it is currently implemented (tick += deltaInMs * 1000;). 

Specifically, I intend as much as possible to place them -within- methods rather than around the method call. But the overarching question is, what should be monitored this way and what shouldn't? And isn't this monitoring the role of a standard profiler anyway rather than the code itself?